### PR TITLE
feat: help command

### PR DESCRIPTION
**Pull request by vohoanglong0107**

### CODE REVIEW

**Review by Kobzol — CHANGES_REQUESTED** [review]
Looks great! One small nit.

File: src/bors/handlers/help.rs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+use crate::bors::RepositoryClient;
+use crate::bors::RepositoryState;
+use crate::github::PullRequest;
+
+const HELP_MESSAGE: &str = r#"
+- try: Request that the PR be tested by buildbot, without accepting it.
```
Comment (Kobzol) [inline src/bors/handlers/help.rs:6]:
```suggestion
- try: Execute the `try` CI workflow on this PR (without approving it for a merge).
```

`buildbot` is actually the name of an existing CI service, which we do not use. This could be a bit confusing to users. We also can't just say to "test this PR on CI", because all PRs are tested on CI, just with different workflows xD So I suggest rewording a bit.

**Review by Kobzol — CHANGES_REQUESTED** [review]
Once you reword the help text, could you please squash the commits? :) Let me know if you want help with that.

File: src/bors/handlers/help.rs (hunk 1 above)
Comment (vohoanglong0107) [reply src/bors/handlers/help.rs:6]: Really sorry about that, I copy it from the old homu page without double checking it

**Review by Kobzol — APPROVED** [review]
Thank you! :heart:
